.travis.yml

```diff
@@ -5,12 +5,18 @@ php:
   - 5.3
   - 5.4
   - 5.5
+  - 5.6
 
 env:
   - SYMFONY_VERSION=2.1.*
   - SYMFONY_VERSION=2.2.*
   - SYMFONY_VERSION=2.3.*
   - SYMFONY_VERSION=2.4.*
+  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION=2.6.*
```

Comment:
I wouldn't add tests for these two versions as they already reached their end of maintenance (2.5 actually even reached its end of life). For the same reason you might want to drop the build for 2.4 too (the builds for 2.1 and 2.2 actually do not make much sense at all as the MobileDetectBundle already requires the version of the FrameworkBundle to be 2.3 or higher).